src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingStyleOptionPageViewModel.cs

```diff
@@ -275,6 +275,34 @@ public bool CanMoveDown
                 }
             }
 
+            public string MoveUpAutomationText
+            {
+                get
+                {
+                    if (!CanMoveUp)
+                    {
+                        return string.Empty;
```

Comment:
We don't always return the same text? The button is disabled, shouldn't the screen reader either be reading nothing, or be at least saying "move up disabled" or something? That seems better than potentially having an unlabeled button.